### PR TITLE
Convert config updating and video/audio queues to use zmq

### LINKED ISSUE #4326: Allow to enable/disable motion detection from Frigate UI and `frigate.yml`
**Describe what you are trying to accomplish and why in non technical terms**
I can disable from Home Assistant:

image

But I can't disable it from the Frigate UI or the configuration file.

image

image

**Describe the solution you'd like**

I believe that, for consistency I should be able to disable it from the configuration file or from the  Frigate UI as well.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by NickM-27**
This PR removes the config fields in the mp.dict and the video/audio output queues in favor of using zero mq. This allows a more straightforward approach to updating data in separate processes.

As a side effect this closes https://github.com/blakeblackshear/frigate/issues/4326 because a motion config is required to hold the current state of motion detection

### CODE REVIEW

File: frigate/config.py [hunk 1]
```diff
@@ -1041,6 +1045,14 @@ def verify_autotrack_zones(camera_config: CameraConfig) -> ValueError | None:
         )
 
 
+def verify_motion_and_detect(camera_config: CameraConfig) -> ValueError | None:
+    """Verify that required_zones are specified when autotracking is enabled."""
+    if camera_config.detect.enabled and not camera_config.motion.enabled:
+        raise ValueError(
+            f"Camera {camera_config.name} has motion detection disabled and object detection enabled but object detection requires motion detection ."
```
Comment (NickM-27) [inline frigate/config.py:1052]:
```suggestion
            f"Camera {camera_config.name} has motion detection disabled and object detection enabled but object detection requires motion detection."
```

**Review by blakeblackshear — APPROVED** [review]

**Comment by regystro** [comment]
> As a side effect this closes #4326 because a motion config is required to hold the current state of motion detection

Sorry, but I don't understand. What do I have to put in the config file to ensure a certain camera starts with motion detection disabled when frigate restarts? I want the camera to not be disabled, but start with motion and detection disabled. Detection can be disabled at startup, but cannot figure out how to start with motion disabled.

Thank you.

**Comment by NickM-27** [comment]
This isn't available in anything but the dev version, you should wait until the release when the docs are updated

**Comment by regystro** [comment]
Cool. Many thanks for the clarification.

**Pull request merged**